spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/DatastoreTemplate.java

```diff
@@ -90,6 +96,8 @@
 
 	private final ObjectToKeyFactory objectToKeyFactory;
 
+	private @Nullable ApplicationEventPublisher eventPublisher;
```

Comment:
What's the purpose of `@Nullable` here?
Also, once the bean is initialized, it should not be null.

Reply:
I saw this from the other Spring Data implementations. I think this is here because there isn't necessarily always a publisher in the context (like if the user just isn't handling events from Spring).